src/registry/addressNL/preview.tsx

```diff
@@ -56,6 +56,26 @@ const Preview: React.FC<ComponentPreviewProps<AddressNLComponentSchema>> = ({com
           />
         }
       />
+      <TextField
+        name={`${key}.city`}
+        label={
+          <FormattedMessage
+            description="Label for addressNL city read only result"
+            defaultMessage="City"
+          />
+        }
+        disabled
+      />
+      <TextField
+        name={`${key}.streetNumber`}
+        label={
+          <FormattedMessage
+            description="Label for addressNL streetName read only result"
+            defaultMessage="Street name"
+          />
+        }
+        disabled
+      />
```

Comment:
These fields should only be displayed when `deriveAddress` is `true`, otherwise not. That applies for both the form builder and the SDK :)